sources/ftypes.h

```diff
@@ -311,6 +311,9 @@ typedef int (*TFUN1)();
 #define NORMALFORMAT 0
 #define NOSPACEFORMAT 1
 
+#define NOCHECKLOGTYPE 0
```

Comment:
I think it's good to add a comment saying they are for `checkLogType` in `WriteStats()`.